### ISSUE TITLE
[Bug] Switch Component: Change state programmatically

### ISSUE THREAD

**Issue by prabh-62** [telerik/kendo-angular#1447]
## I'm submitting a...

* Bug report

## Current behavior
In order to change the state of switch Component programmatically, I considered the following four choices
1. 
```typescript
@Component({
    selector: 'my-app',
    template: `
        <kendo-switch
          [(ngModel)]="checked"
        ></kendo-switch>
    `
})

class AppComponent {
    public checked: boolean = true;
}
```
doesn't work if <kendo-switch> is nested inside 
```html
<div [formGroup]="someForm">...</div
```

2.

```html
 <kendo-switch [checked]="false"></kendo-switch>
```
```typescript
    public checked: boolean = true;
// and then toggle the checked variable
}
```

3. 
```typescript
this.kendoSwitchComponent.changeValue(false)
```
 doesn't drag switch to either side

4. Hack
```typescript
      const spaceKey = new KeyboardEvent('keydown', {
        keyCode: '32'
      });
      return this.kendoSwitchComponent.keyDown(spaceKey);
```

## Expected behavior

On Changing the value programmatically, Switch toggle doesn't slide but does change the state

## Minimal reproduction of the problem with instructions
Plunker: https://plnkr.co/edit/IoB0sLXCM3fpziYDIrPC?p=preview

## Environment

Package versions:
```
> npm ls --depth 0
+-- @angular/animations@5.2.9
+-- @angular/cli@1.6.7
+-- @angular/common@5.2.9
+-- @angular/compiler@5.2.9
+-- @angular/compiler-cli@5.2.9
+-- @angular/core@5.2.9
+-- @angular/forms@5.2.9
+-- @angular/language-service@5.2.9
+-- @angular/platform-browser@5.2.9
+-- @angular/platform-browser-dynamic@5.2.9
+-- @angular/router@5.2.9
+-- @progress/kendo-angular-buttons@4.0.0
+-- @progress/kendo-angular-dateinputs@2.2.0
+-- @progress/kendo-angular-dialog@3.4.0
+-- @progress/kendo-angular-dropdowns@2.1.0
+-- @progress/kendo-angular-excel-export@1.0.7
+-- @progress/kendo-angular-grid@3.0.0
+-- @progress/kendo-angular-inputs@2.2.0
+-- @progress/kendo-angular-intl@1.4.0
+-- @progress/kendo-angular-l10n@1.1.0
+-- @progress/kendo-angular-layout@2.2.1
+-- @progress/kendo-angular-resize-sensor@3.0.2
+-- @progress/kendo-angular-ripple@0.2.5
+-- @progress/kendo-angular-scrollview@1.0.11
+-- @progress/kendo-angular-sortable@1.0.8
+-- @progress/kendo-data-query@1.2.2
+-- @progress/kendo-drawing@1.5.2
+-- @progress/kendo-theme-default@2.49.1
+-- rxjs@5.5.8
+-- ts-node@5.0.1
+-- tslint@5.9.1
+-- typescript@2.6.2
`-- zone.js@0.8.25
```

**Comment by danielkaradachki** [comment]
The problem with the checked property binding should be fixed in the latest [dev](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) version:
https://plnkr.co/edit/0uBNcy3enddUfpgsntmv?p=preview

I am not sure if I understand the issue with ngModel but you will get an error for any input if you nest ngModel inside formGroup:
https://plnkr.co/edit/DxMu0tmNMSVPA8m91HTX?p=preview
In general, template and reactive forms should not be mixed.

changeValue and keyDown are internal methods and should not be used.

**Comment by tsvetomir** [comment]
Fixed in v3.0.0
